content/management/configuration-api/v3.2/index.mdx

```diff
@@ -32,6 +32,720 @@ Summing up, if the user token starts with `fra-`, you should add the `X-Region:
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### Update Agent
+
+Updates the properties of an Agent specified by `id`.
+
+#### Specifics
+
+|                 |                                                                                   |
+|-----------------|-----------------------------------------------------------------------------------|
+| Method URL      | `https://api.livechatinc.com/v3.2/configuration/action/update_agent`              |
+| Required scopes | `agents--my:rw` `agents--all:rw` (to update other Agents within the same license) |
+
+#### Request
+
+| Parameter                                | Data type  | Required | Notes                                                                         |
+|------------------------------------------|------------|----------|-------------------------------------------------------------------------------|
+| `id`                                     | `string`   | Yes      | Agent ID                                                                      |
+| `name`                                   | `string`   | No       | Agent name                                                                    |
+| `role`                                   | `string`   | No       | Agent role, should be on of following: `viceowner`, `administrator`, `normal` |
+| `avatar`                                 | `string`   | No       | URL path for agent avatar                                                     |
+| `job_title`                              | `string`   | No       | Agent's job title                                                             |
+| `mobile`                                 | `string`   | No       | Agent's mobile number                                                         |
+| `max_chats_count`                        | `int`      | No       | Agent's maximum number of concurrent chats                                    |
+| `groups`                                 | `object[]` | No       | Groups an Agent belongs to.                                                   |
+| `groups[].id`                            | `uint`     | Yes      | Group ID; required only when `group`'s included.                              |
+| `groups[].priority` __*****__            | `string`   | Yes      | Agent's priority in a group; required only when `group`'s included.           |
+| `notifications`__***__                   | `string[]` | No       | Represents which agent notifications are turned on.                           |
+| `email_subscriptions`__****__            | `string[]` | No       | Represents which subscriptions will be send to agent via email.               |
+| `work_scheduler`                         | `object`   | No       | Work scheduler options to set for new agent                                   |
+| `work_scheduler.__**__.<work_scheduler>` | `object`   | No       | **\**\*** **described below**                                                 |
+
```

Comment:
Please, make sure the order and the number of * is correct, as for the one we discussed via DMs.